Number_types/include/CGAL/Mpzf.h

```diff
@@ -909,7 +917,11 @@ struct Mpzf {
   Mpzf& operator+=(Mpzf const&x){ *this=*this+x; return *this; }
   Mpzf& operator-=(Mpzf const&x){ *this=*this-x; return *this; }
   Mpzf& operator*=(Mpzf const&x){ *this=*this*x; return *this; }
+#ifdef CGAL_MPZF_DIVISION_OPERATOR
   Mpzf& operator/=(Mpzf const&x){ *this=*this/x; return *this; }
+#else // not CGAL_MPZF_DIVISION_OPERATOR
+  Mpzf& operator/=(Mpzf const&x){ *this=division(*this,x); return *this; }
```

Comment:
Shouldn't `operator/=` be disabled when we disable `operator/`?

Reply:
Hi Marc. You seem right. Would you mind write the pull-request?